tasks/transifex-resjson-tasks.js

```diff
@@ -720,7 +720,7 @@ module.exports = function (grunt) {
         undefined if the code isn't of correct format.
     */
     function mapToTxLangCode(str) {
-        var matcher = str.match(/([a-z]{2})-([A-Z]{2}|latn)$/i);
+        var matcher = str.match(/([a-z]{2,3})(?:[-_]([a-z]{2}|latn))?$/i);
```

Comment:
I thought it would not hurt to accept the right format directly (so that people can name their folder after Transifex's nomenclature if they prefer (e.g. `en_US`)?